include/vault.bash

```diff
@@ -231,6 +233,12 @@ cloudbreak-delete-vault-data() {
 
     remove-variable-from-profile VAULT_UNSEAL_KEYS
     remove-variable-from-profile VAULT_ROOT_TOKEN
-    
+
     rm -rf $VAULT_CONFIG_FILE
 }
+
+create-token-file() {
+    mkdir -p ./etc
+    # token file is needed for Audit Service with a UUID inside it. This is temporary until Thunderhead codebase is updated to accept real root tokens.
+    uuidgen > "./etc/vault-dev-root-token.txt"
```

Comment:
How is this going to work with Vault?

Reply:
NVM. I see your comment.

Reply:
It won't work with our Vault as of now. This file is just needed for Audit Service to boot up.